src/mantine-core/src/components/RadioGroup/Radio/Radio.styles.ts

```diff
@@ -70,14 +70,19 @@ export default createStyles((theme, { size, color }: RadioStyles, getRef) => {
     label: {
       ...theme.fn.fontStyles(),
       display: 'flex',
-      alignItems: 'center',
+      alignItems: 'flex-start',
```

Comment:
This is still fine and centered for one-line labels:
![image](https://user-images.githubusercontent.com/9464690/157602065-f2bf85ab-eec5-4395-b612-9a5682fcc223.png)